test/WebSites/MvcTagHelpersWebSite/wwwroot/site.css

```diff
@@ -0,0 +1,2 @@
+﻿body {
```

Comment:
To make it easy for a person debugging this functional test to see the output you should fill each one of these `site.css` files with something specific that shows on the functional test page. This way the tests that you provide actually do something that's added by these css files.